src/api/pastel-rpc/network-stats.ts

```diff
@@ -0,0 +1,506 @@
+/* eslint-disable */
+import {
+  BlockInfo,
+  BlockSubsidy,
+  ChainTips,
+  ListTransactions,
+  ListUnspent,
+  MempoolInfo,
+  MiningInfo,
+  NetTotals,
+  NetworkInfo,
+  PastelRPCResponse,
+  RawTransaction,
+  ReceivedByAddress,
+  TotalBalance,
+  TransactionInfo,
+  TxoutsetInfo,
+  WalletInfo,
+} from '../../features/type'
+import { rpc, TRPCConfig } from './rpc'
+
+interface StatisticType {
+  hashrate: string
+  difficulty: string
+  block: string
+}
+
+export async function getStatisticInfos(
+  config: TRPCConfig,
+): Promise<StatisticType> {
+  try {
+    const hashrate = await getNetworkHashps(config)
+    const difficulty = await getDifficulty(config)
+    return {
+      hashrate: hashrate,
+      difficulty: difficulty,
+      block: '',
+    }
+  } catch (error) {
+    return {
+      hashrate: '',
+      difficulty: '',
+      block: '',
+    }
+  }
+}
+
+export async function getNetworkHashps(config: TRPCConfig): Promise<string> {
+  const res: PastelRPCResponse = await rpc('getnetworkhashps', [], config)
```

Comment:
This should be
```js
return rpc<PastelRPCResponse>('getdifficulty', [], config)
```
But why do you think that `PastelRPCResponse` is the format that comes back from RPC? I'm more than sure that it's not. 

Every method in RPC returns a different result. You need to figure out what is the return of each function and fix all of them